volestipy/tests/test_svd_rounding.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python3.6
+
+import numpy as np
+import volestipy
+from volestipy import HPolytope
+
+if __name__ == "__main__":
+    dim = 5
+    A = np.zeros((2*dim, dim), dtype=np.float)
+    A[0:dim] = np.eye(dim)
+    A[dim:] -=  np.eye(dim,dim, dtype=np.float)
+    b = np.ones(2*dim, dtype=np.float)
+
+
+    hp = HPolytope(A,b)
+
+#    print("The polytope:")
```

Comment:
please remove if not needed or print in debug more by using some type of flag

Reply:
Changed the test to be more "elegant"